src/emmc.c

```diff
@@ -231,6 +234,12 @@ pu_emmc_setup_layout(PuFlash *flash,
 
     ped_disk_commit(self->disk);
 
+    udevadm_cmd = g_strdup_printf("udevadm settle --timeout %d", UDEVADM_SETTLE_TIMEOUT);
+    if (!g_spawn_command_line_sync(udevadm_cmd, NULL, NULL, &wait_status, error))
+        return FALSE;
+    if (!g_spawn_check_exit_status(wait_status, error))
+        return FALSE;
+
```

Comment:
Maybe it's worth separating this into its own function. Although there is currently only one place where it is used, it makes sense from a logical point of view.

Reply:
Sure, I'll move it to utils.